src/Tools/dotnet-monitor/Strings.resx

```diff
@@ -930,8 +930,8 @@
   <data name="ParameterCapturingNotAvailable_Reason_PreventedHostingStartup" xml:space="preserve">
     <value>The process has prevented hosting startup assemblies from loading using the ASPNETCORE_PREVENTHOSTINGSTARTUP environment variable.</value>
   </data>
-  <data name="ParameterCapturingNotAvailable_Reason_HostingStartupDidNotLoad" xml:space="preserve">
-    <value>This feature is only available on processes using ASP.NET Core. If the process is using ASP.NET Core and has successfully started, ensure that it has not been configured to prevent hosting startup assemblies from loading.</value>
+  <data name="ParameterCapturingNotAvailable_Reason_StartupHookDidNotLoad" xml:space="preserve">
+    <value>This feature requires that the DOTNET_STARTUP_HOOKS environment variable is set on the target process.</value>
```

Comment:
This explanation isn't really correct. In .NET 8+, we don't use that environment variable to load the startup hook but use a diagnostic IPC command to dynamically load it. Maybe just say "This feature requires the startup hook to be loaded in the target process." That should be sufficient since I believe the `LogFormatString_StartupHookApplyFailed` string will be logged in .NET Monitor's output shortly before `ErrorMessage_ParameterCapturingNotAvailable` string is logged.